nextcord/types/appinfo.py

```diff
@@ -40,30 +43,23 @@ class BaseAppInfo(TypedDict):
     description: str
 
 
-class _AppInfoOptional(TypedDict, total=False):
-    team: Team
-    terms_of_service_url: str
-    privacy_policy_url: str
-    hook: bool
-    max_participants: int
-
-
-class AppInfo(BaseAppInfo, _AppInfoOptional):
+class AppInfo(BaseAppInfo):
+    team: NotRequired[Team]
+    terms_of_service_url: NotRequired[str]
+    privacy_policy_url: NotRequired[str]
+    hook: NotRequired[bool]
+    max_participants: NotRequired[int]
     rpc_origins: List[str]
     owner: User
     bot_public: bool
     bot_require_code_grant: bool
 
 
-class _PartialAppInfoOptional(TypedDict, total=False):
-    rpc_origins: List[str]
-    cover_image: str
-    hook: bool
-    terms_of_service_url: str
-    privacy_policy_url: str
-    max_participants: int
-    flags: int
-
-
-class PartialAppInfo(_PartialAppInfoOptional, BaseAppInfo):
-    pass
+class PartialAppInfo(BaseAppInfo):
```

Comment:
I'm not sure if this would affect the inherited fields, but couldn't you use `total=False` here?